apps/wallet/src/background/keyring/Account.ts

```diff
@@ -1,66 +1,54 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { type SuiAddress } from '@mysten/sui.js';
+
+import {
+    type DerivedAccount,
+    type SerializedDerivedAccount,
+} from './DerivedAccount';
+import {
+    type ImportedAccount,
+    type SerializedImportedAccount,
+} from './ImportedAccount';
 import {
-    normalizeSuiAddress,
-    toSerializedSignature,
-    type SerializedSignature,
-    type Keypair,
-    type SuiAddress,
-} from '@mysten/sui.js';
+    type LedgerAccount,
+    type SerializedLedgerAccount,
+} from './LedgerAccount';
+
+export enum AccountType {
+    IMPORTED = 'imported',
```

Comment:
I personally try to make keys and values match here to make debugging a little easier.